src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -183,6 +183,8 @@ public function caution($message)
      */
     public function table(array $headers, array $rows)
     {
+        $headers = array_map(function ($value) { return sprintf('<info>%s</>', $value); }, $headers);
```

Comment:
Just out of interest: what's the advantage of `</>`? I'm seeing it more and more these days and I'm not sure why. Probably, there is a reason for it.